_includes/integrations/angular-tech-ref.md

```diff
@@ -2,6 +2,7 @@
 
 **Covered in this section:**
 
+- [Supported Angular versions](#supportangularversion)
```

Comment:
```suggestion
- [Supported Angular versions](#supportedangularversions)
```

Reply:
I assume this needs to match the heading to work?

Reply:
yeah, they do :) Thanks